tensorflow_datasets/object_detection/malaria_infected_smears.py

```diff
@@ -0,0 +1,149 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Malaria Infected Human Blood Smears Dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
Please remove 

> from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

as we dropped use of python 2 so remove all unnecessary code